packages/patternlab/styleguide/source/_patterns/03-organisms/by-author/inline-links.md

```diff
@@ -0,0 +1,20 @@
+### Description
+Renders an inline list of links or text items (if URL is not provided for the link).
+It provides an alternative background using the "background" variable.
+
+
+### Variables
+~~~
+inlineLinks: {
+  ariaLabel: string / optional
+  background: [{null, "alternative"}] / optional
```

Comment:
```suggestion
  background: [{null, "alternative"}] / optional
```
instead of "alternative" can you use label like "gray"? or `background: true` I think that's more inline with other components and more intuitive.

Reply:
done